packaging/imagemode/Makefile

```diff
@@ -0,0 +1,139 @@
+		--quiet \
+		--queryformat '%{version}' \
+		--disablerepo '*' \
+		--repofrompath "this,${USHIFT_URL}" \
+		--latest-limit 1 | sed 's/~/-/g') ; \
+	IMAGE_NAME="microshift-$${IMAGE_VER}" ; \
+	sudo podman build \
+		--no-cache \
+		--authfile "${PULL_SECRET}" \
+		--build-arg BASE_IMAGE_URL=${BASE_IMAGE_URL} \
+		--build-arg BASE_IMAGE_TAG=${BASE_IMAGE_TAG} \
+		--build-arg USHIFT_VER=$${IMAGE_VER} \
+		--build-arg USHIFT_URL=$${USHIFT_URL} \
+		--build-arg OCPDEP_URL=$${OCPDEP_URL} \
+		-t "$${IMAGE_NAME}" \
+		-f Containerfile.repourl
+
+run:
+ifndef IMAGE_VER
+	$(error Run 'make run IMAGE_VER=value')
+endif
+	IMAGE_NAME="microshift-${IMAGE_VER}" ; \
+	sudo modprobe openvswitch ; \
+	CONT_ID=$$(sudo podman run --rm -d --privileged \
+		-v "${PULL_SECRET}":/etc/crio/openshift-pull-secret:ro \
+		-v /var/lib/containers/storage:/var/lib/containers/storage \
+		--replace \
+		--name "$${IMAGE_NAME}" \
+	    "$${IMAGE_NAME}") ; \
+	echo "sudo podman exec -it $${CONT_ID} bash"
+
+stop:
+	CONT_IDS=$$(sudo podman ps --format "{{.Names}}" | grep '^microshift-' || true) ; \
+	if [ -n "$${CONT_IDS}" ] ; then sudo podman stop $${CONT_IDS} ; fi
+
+clean:
+	sudo podman rmi -af
+
+#
+# Predefined container image builds
+#
+.PHONY: 4.16-el94
+4.16-el94:
+	@$(MAKE) rhocp USHIFT_VER=4.16
+
+.PHONY: 4.17-rc-el94
+4.17-rc-el94:
+	@$(MAKE) repourl \
+		USHIFT_URL=https://mirror.openshift.com/pub/openshift-v4/$$(uname -m)/microshift/ocp/latest-4.17/el9/os/ \
+		OCPDEP_URL=https://mirror.openshift.com/pub/openshift-v4/$$(uname -m)/dependencies/rpms/4.17-el9-beta/
+
+.PHONY: 4.18-ec-el94
+4.18-ec-el94:
+	@$(MAKE) repourl \
+		USHIFT_URL=https://mirror.openshift.com/pub/openshift-v4/$$(uname -m)/microshift/ocp-dev-preview/latest-4.18/el9/os/ \
+		OCPDEP_URL=https://mirror.openshift.com/pub/openshift-v4/$$(uname -m)/dependencies/rpms/4.18-el9-beta/
+
+#
+# ISO builds are named 'microshift-<make_target>' (the '.iso' suffix is part of
+# the make target name) and stored at the ${BIB_OUTBASE} directory.
+# Each build uses 'localhost/microshift-<make-target>' container image as input.
+#
+# BIB_IMAGE_URL ?= registry.redhat.io/rhel9/bootc-image-builder
+# BIB_IMAGE_TAG ?= latest
+# BIB_OUTBASE ?= $(HOME)/imagemode
+# .PHONY: 4.16-el94.iso
+# 4.16-el94.iso: 4.16-el94
+# 	IMAGE_NAME="microshift-$$(echo $@ | sed 's/\.iso$$//')" ; \
+# 	ISO_FNAME="microshift-$@" ; \
+# 	BIB_OUTDIR="${BIB_OUTBASE}/$@" ; \
+# 	sudo podman pull \
+# 		--authfile "${PULL_SECRET}" \
+# 		"${BIB_IMAGE_URL}:${BIB_IMAGE_TAG}" && \
+# 	sudo mkdir -p "$${BIB_OUTDIR}" && \
+# 	sudo podman run \
+# 		--rm -i --privileged \
+# 		--pull=never \
+# 		--security-opt "label=type:unconfined_t" \
+# 		-v "$${BIB_OUTDIR}:/output" \
+# 		-v "/var/lib/containers/storage:/var/lib/containers/storage" \
+# 		"${BIB_IMAGE_URL}:${BIB_IMAGE_TAG}" \
+# 		--type anaconda-iso \
+# 		--local \
+# 		"localhost/$${IMAGE_NAME_URL}" && \
+# 	sudo mv -f "$${BIB_OUTDIR}/bootiso/install.iso" "${BIB_OUTBASE}/$${ISO_FNAME}"
```

Comment:
I presume this will be doable in near future and we'll uncomment it?

Reply:
Yes, our follow-up project would be to create ISO images from the container images.
So, these are the commands we can use. Do you think it's a problem to merge this code in its current form?
I can remove those.

Reply:
BTW, it is already doable now. I just did not want to create unnecessary "noise" in the code and docs.